docs/sources/dashboards/build-dashboards/create-dashboard/index.md

```diff
@@ -61,6 +61,10 @@ Dashboards and panels allow you to show your data in visual form. Each panel nee
 
    For more information about individual visualizations, refer to [Visualizations options][].
 
+1. Under **Panel options**, enter a title and description for your panel.
+
+   Alternatively, Grafana can generate a panel title and description for you using the OpenAI integration. Learn more in the [Set up generative AI features for dashboards documentation][].
```

Comment:
@SandersAaronD thoughts on saying `OpenAI integration` explicitly vs `LLM integration`? Not sure if we want to be LLM agnostic 🤷

Reply:
@nmarrs & @SandersAaronD That was a bit of a leftover from the original notes that I took. I'm happy to change that to "LLM integration".